src/main/java/com/tstepnik/planner/domain/Task.java

```diff
@@ -16,13 +19,36 @@
     @Enumerated(EnumType.STRING)
     private Importance importance;
 
-    private boolean isDone;
+    private Boolean isDone;
 
     private Long userId;
 
+    private ZonedDateTime creationDate;
```

Comment:
Never used this type of date time. Why not use LocalDateTime?

Reply:
What are the advantages of this type and why did you decide to use it?

Reply:
Because of date format and time zone, with LocalDateTime I have gotten "local" time in Warsaw/Poland and i needed UTC.

Reply:
https://thorben-janssen.com/hibernate-jpa-date-and-time/

Reply:
I was looking for solution to set UTC time, but i figure that LocalDateTime also have possibility to set it.

Reply:
Without ZonedDateTime I cannot set HH:MM to UTC zone. I try do it with LocalDateTime but always get -2 hour in databese.

Reply:
Time showed in the app will be different than time in the database. We can easily use LocalDateTime (that is what we always use) and we just need to make sure that database stores time in UTC format